types/opentype.js/index.d.ts

```diff
@@ -167,48 +167,48 @@ export interface Field {
 
 export class Glyph {
     index: number;
-    private xMin;
-    private xMax;
-    private yMin;
-    private yMax;
-    private points;
-
-    name: string;
-    path: Path | (() => Path);
-    unicode: number;
+    name: string | null;
+    unicode?: number | undefined;
     unicodes: number[];
-    advanceWidth: number;
+    xMin?: number | undefined;
+    xMax?: number | undefined;
+    yMin?: number | undefined;
+    yMax?: number | undefined;
+    advanceWidth?: number | undefined;
+    leftSideBearing?: number | undefined;
+    path: Path;
 
+    private bindConstructorValues(options: GlyphOptions): void;
```

Comment:
There's no point in calling this outside of the constructor. The library itself never uses it either.

Reply:
there as no point of addition 'private', just remove from declaration, so user cannot access this (unless doing module augmentation)

Reply:
Tried to google this when I was creating the PR, but couldn't find anything conclusive. My intention was to document this for future contributors, so that they don't add this back by mistake. Or is a comment preferred?

Reply:
easy, improve in future, it will come back